src/openforms/conf/base.py

```diff
@@ -929,38 +891,21 @@
                 {"set_number": "1", "name": "urn:etoegang:1.9:EntityConcernedID:KvKnr"},
                 {"set_number": "2", "name": "urn:etoegang:1.9:EntityConcernedID:KvKnr"},
             ],
-            "requested_attributes": [
-                "urn:etoegang:1.11:attribute-represented:KvKnr",
-            ],
-            "privacy_policy_url": {
-                "nl": PRIVACY_POLICY_URL,
-            },
-            "herkenningsmakelaars_id": EHERKENNING_MAKELAAR_ID,
+            "requested_attributes": [],
+            "language": "nl",
         },
         {
```

Comment:
Can we add an IF-statement to add the eIDAS part below dynamically? For example:

```python
if EIDAS_SERVICE_INDEX:
  EHERKENNING["services"].append(....)
```

Or does this cause other issues?

Reply:
I think it should be fine